### PR TITLE
Implement browsing by tag

### PR DESCRIPTION
**Pull request by stesh**
#10 ye

### CODE REVIEW

File: porick/views.py [hunk 1]
```diff
@@ -6,8 +6,14 @@
...
-from .models import Quote, User, AREA_ORDER_MAP, DEFAULT_ORDER, QSTATUS
-
+from .models import (
+    AREA_ORDER_MAP,
+    DEFAULT_ORDER,
+    QSTATUS,
+    Quote,
+    Tag,
+    User,
+)
```
Comment (kopf) [inline porick/views.py:16]: hngh

**Comment by kopf** [comment]
The list of tags page `/tags` is missing bro

**Comment by stesh** [comment]
Will implement that in a separate PR.

**Comment by kopf** [comment]
ok, don't see the sense in that, since the `/tags` route will then be broken, but anyhow

**Comment by stesh** [comment]
Both routes are broken in master!

**Comment by kopf** [comment]
Not getting into an argument here, just pointing something out!

Both routes were not implemented in master, and both raised a `NotImplementedError`. This was an accurate reflection of the state of the code, and not "broken".

Now, one route is working, and one is broken, but there's no way for a third party (say, @johnlunney) to see that one of them is broken by glancing over the code. It looks as if both routes are implemented. 

That's why it's nicer if we work on each feature from beginning to completion in a single branch and merge in a single pull request. :)

**Pull request merged**